plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3967,6 +3974,7 @@ blockly.warning.place_feature_ghost_blocks=Some features might not be placed due
 blockly.warning.lake_feature_crash=Due to a bug in worldgen code, lake features containing water will cause the game to crash during world generation.
 blockly.warning.state_provider_int_property_warning=Data pack validation will FAIL if the block doesn''t have the given property, or if the value is out of bounds.
 blockly.warning.utility_adv_trigger=Trigger {0} is a utility advancement trigger and should not be used in combination with other triggers.
+blockly.warning.first_component_type_only=Each type of component only generates the first block of the type. The following blocks will be skipped until a new component type is found.
```

Comment:
It would be neat to see Blockly validating that only one type of each type being present instead of emitting warning in all cases for UX reasons